packages/mongo/mongo_driver_tests.js

```diff
@@ -0,0 +1,9 @@
+const MongoDB = NpmModuleMongodb;
```

Comment:
Instead of accessing `MongoDB` this way, could you follow the approach that's currently used with some of the [other tests](https://github.com/meteor/meteor/blob/5e9ff342a8ce9e488e2af4086d39ac1e73325e9c/packages/mongo/mongo_livedata_tests.js#L3237): `const MongoDB = MongoInternals.NpmModules.mongodb.module;` This helps keep things more uniform.